agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -154,6 +154,16 @@ void slave_thread::slave_reset()
     is_backhaul_manager   = false;
     detach_on_conf_change = false;
 
+    auto db = AgentDB::get();
+
+    auto radio = db->radio(m_fronthaul_iface);
+    if (!radio) {
+        LOG(DEBUG) << "Radio of iface " << m_fronthaul_iface << " does not exist on the db";
+        return;
+    }
```

Comment:
The rest of the function should not be performed because it will be depended on the fact that we will get the radio struct from the database.
It should not happen obviously, and if it happens the son_slave will be stuck on some state trying to restart again and again, which is exactly what needs to be done, inorder it will be fixed.
Anyway I changed the print type tp `ERROR`